pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -85,6 +85,11 @@ func checkPortAvailable(
 		// XXX Fallback until we have complete Name support in UI
 		portStatus = ctx.deviceNetworkStatus.GetPortByIfName(status.Port)
 		if portStatus == nil {
+			// internal airgap switch case
+			if status.Port == "0" {
```

Comment:
No, that bug in the UI needs to be fixed.
EVE code shouldn't have random workarounds for broken controller/UI code.